ManageWikiSettings.php

```diff
@@ -766,6 +766,15 @@
 		'help' => 'Page titles that are not allowed to be rated.',
 		'requires' => [],
 	],
+	'wgRPAddSidebarSection' => [
+		'name' => 'Rate Page Add Sidebar Section',
+		'from' => 'ratepage',
+		'type' => 'check',
+		'overridedefault' => true,
+		'section' => 'editing',
+		'help' => 'whether to add the default sidebar section for the widget',
```

Comment:
Minor, but can this be capitalised first letter?

Reply:
Done, thanks!